data/data/gcp/network/network.tf

```diff
@@ -50,9 +53,12 @@ resource "google_compute_router_nat" "worker_nat" {
   name                               = "${var.cluster_id}-nat-worker"
   router                             = google_compute_router.router[0].name
   nat_ip_allocate_option             = "AUTO_ONLY"
-  min_ports_per_vm                   = 512
+  min_ports_per_vm                   = 7168
```

Comment:
If this setting is identical for master/worker now, is there still value in having two nat gateways?

Reply:
Good question. I would say stick to keeping two separate ones. Once a proper investigation is done into all of this we might realize we need to separate values. But then again, I am always in favor of the philosophy "fix it when broken/do it when needed", but in this case I am not sure. Maybe someone from installer has a stronger opinion?